sources/packages/backend/libs/test-utils/src/mocks/ssh-service-mock.ts

```diff
@@ -50,6 +50,11 @@ export function getUploadedFile(
   sshClientMock: DeepMocked<Client>,
 ): UploadedFile {
```

Comment:
Not a part of this PR but maybe we can update the comment to: "Get the parameters provided to the put method of the SSH client that represents **the remote file path and** the data that would be uploaded to the SFTP in a real scenario."

Reply:
Changed.